mne/io/brainvision/brainvision.py

```diff
@@ -263,7 +263,8 @@ def _read_vmrk_events(fname, event_id=None, response_trig_shift=0):
         txt = txt.decode('latin-1')
 
     # extract Marker Infos block
-    m = re.search(r"\[Marker Infos\]", txt)
+    m = re.search(r"\[Marker Infos\]", txt,re.IGNORECASE)
```

Comment:
This will elicit a PEP8 error, need a space between args/comma as `txt, re.IGNORECASE`

Reply:
Sorry forgot the space will add that and the data shortly, do you want me to add some asserts in the test_brainvision.py file too?

Reply:
Please add the data to the `mne-testing-data` repo, not `mne-python`. After that is in, you can follow the instructions:

https://github.com/mne-tools/mne-testing-data#add-or-change-files-in-the-repo-for-use-with-mne

Then update a test here